cogs/cipher.py

```diff
@@ -0,0 +1,129 @@
+import base64
+import hashlib
+import re
+import string
+
+from discord.ext import commands
+
+import cogs.utils.parser as parser
+from assets.context import Command, argument, example, usage_info
+
+
+class Ciphers(commands.Cog):
+    character_set = [string.ascii_lowercase, string.ascii_uppercase, string.digits]
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def cog_check(self, ctx):
+        if self.bot.is_ready():
+            return True
+        else:
+            return False
+
+    @commands.command(cls=Command,
+                      description="Shifts the letters within a message",
+                      syntax=[argument(True, "Shift defaults to 3"), argument(True, "Message")],
+                      usage_information=[
+                          usage_info("", "If a shift value is not found at the start "
+                                         "of the message it would try and find it at the end")
+                      ],
+                      examples=[argument("Hello", "Returns Khoor"), argument("10 Hello", "Returns Rovvy")])
```

Comment:
The canonical way of putting a description in a command is with a docstring, not the description kwarg